rvs/conf/nv32/gst_single.conf

```diff
@@ -55,7 +55,7 @@ actions:
   count: 1
   duration: 10000
   copy_matrix: false
-  target_stress: 7000
+  target_stress: 6000
```

Comment:
To be in tune with Navi32 based on trail and error ?

Reply:
yes, mostly this was having some issues in SRIOV setup, and all feature enabled for sriov setup is gonna happen in later release. this one is with some trial and error and some calculation thrown in :)